components/fms-forwarder/Cargo.toml

```diff
@@ -38,6 +39,7 @@ fms-proto = { workspace = true }
 influx-client = { workspace = true, features = ["writer"] }
 log = { workspace = true }
 paho-mqtt = { version = "0.12", default-features = false, features = [ "vendored-ssl" ] }
+zenoh = {version = "0.10.1-rc"}
```

Comment:
will this only work with this particular `rc` version or can we also specify e.g. `0.10` as the requirement?

Reply:
For the time being it works with particular rc version.